apps/opt.c

```diff
@@ -12,6 +12,9 @@
 # include OPENSSL_UNISTD
 #endif
 
+#if !defined(_WIN32)
```

Comment:
This is overly bold statement. I.e. that *every* non-Windows has libgen.h and corresponding functions. This can't be true. And I can imagine that even on Unices its availability (or rather visibility of functions in question) can be conditional, i.e. depend on compiler flags...